PresentScreenings/Controllers/FilmInfoDialogController.cs

```diff
@@ -153,7 +155,17 @@ private void CreateFilmTitleLabel()
         private void CreateScreeningsScrollView()
         {
             // Get the screenings of the selected film.
-            var screenings = ViewController.FilmScreenings(_film.FilmId);
+            var screenings = new List<Screening> { };
+            var programIds = _film.FilmInfo.CombinationProgramIds;
+            if (programIds.Count == 0)
+            {
+                screenings = _film.FilmScreenings;
+            }
+            else
+            {
+                var program = ViewController.GetFilmById(programIds.First());
+                screenings = program.FilmScreenings;
+            }
```

Comment:
Please consider to select all screenings of all combination programs in which this film is screened.

Reply:
Done.